src/Native/build.cmd

```diff
@@ -18,10 +18,10 @@ set MKL_LIB_PATH=""
 :Arg_Loop
 if [%1] == [] goto :ToolsVersion
 if /i [%1] == [Release]     ( set CMAKE_BUILD_TYPE=Release&&shift&goto Arg_Loop)
-if /i [%1] == [Release-Intrinsics]     ( set CMAKE_BUILD_TYPE=Release-Intrinsics&&shift&goto Arg_Loop)
+if /i [%1] == [Release-netcoreapp3_0]     ( set CMAKE_BUILD_TYPE=Release-netcoreapp3_0&&shift&goto Arg_Loop)
```

Comment:
Do we really need 1 build for each configuration? Or can we just use the multi targets and do it all in one build?

Reply:
We build the same product assemblies in each configuration.

But the different configurations change which TFM we run our tests for. We want to run the tests in parallel across the different machines in order to keep CI times down.